jsontableschema/types.py

```diff
@@ -73,16 +73,16 @@ def _check_constraint_list(self, value, constraint_list):
     def cast(self, value, skip_constraints=False):
         """Return boolean if `value` can be cast as type `self.py`."""
 
-        # We check on `constraints.required` before we cast
-        if not skip_constraints:
-            required = self._get_constraint_value('required')
-            if required is not None:
-                if not required and (value in (None, utilities.NULL_VALUES)):
-                    return None
-                elif required and value in (None, ''):
-                    raise exceptions.ConstraintError(
-                        msg="The field '{0}' requires a value".format(
-                            self.field_name))
+        # Return None/raise constraint error if value is NULL_VALUE
+        if value in ([None] + utilities.NULL_VALUES):
+            if not skip_constraints:
```

Comment:
Not sure what's the expected behaviour here, but I think that a required string field getting '' should return '' and not throw an exception.
@pwalsh wdyt?